src/commands/commands.ts

```diff
@@ -0,0 +1,10 @@
+import { CommandInteraction, SlashCommandBuilder } from 'discord.js';
+
+export const gratitude = {
+    data: new SlashCommandBuilder()
+        .setName('gratitude')
+        .setDescription('Tell us what are u grateful for!'),
+    async execute(interaction: CommandInteraction) {
+        await interaction.reply('Pong!');
```

Comment:
@kudziajaroslaw98 Pong 🏓 (chyba też `todo: finish later` cnie?)

Reply:
To bug, execute nie powinno tam być :)